src/explore-education-statistics-common/src/modules/table-tool/components/LocationFiltersForm.tsx

```diff
@@ -47,7 +48,10 @@ const LocationFiltersForm = ({
 
   const stepHeading = (
     <WizardStepHeading {...stepProps} fieldsetHeading>
-      Choose locations
+      {Object.keys(options).length === 1 &&
+      locationLevelsMap[Object.keys(options)[0]]
+        ? `Choose ${locationLevelsMap[Object.keys(options)[0]].plural}`
```

Comment:
Would suggest extracting out the `Object.keys(options)` into a `levelKeys` variable to tidy this up a bit

Reply:
Done